topics/migration.md

```diff
@@ -0,0 +1,307 @@
+---
+title: Migration from Redis to Valkey
+linkTitle: Migration
+description: How to migrate from Redis to Valkey
+---
+
+This is a migration guide from Redis Community Edition to Valkey. You will learn how to migrate a standalone Redis server instance and a Redis cluster. 
+
+This guide provides migration steps for Redis server and Valkey deployed in Docker; however, they should also apply for on-premises deployments. Refer to [install Valkey](installation.md) for installation options.
+
+Redis Community Edition (CE) is a high-performance, in-memory key/value data structure store. Keys are unique identifiers that can be any of the supported data types, such as strings, hashes, lists, sets, sorted sets, and JSON.
+
+Valkey is an open source, in-memory data structure store designed to be a drop-in replacement for Redis Community Edition 7.2.4. It aims to provide enhanced performance, scalability, and additional features while maintaining compatibility with Redis commands and data structures.
+
+## Why to migrate to Valkey?
+
+* Licensing freedom and no vendor lock-in with the open-source software
+* Enhanced performance with multi-threading, dual-channel replication, new I/O threading
+* Scalability via automatic cluster failover and improved scaling
+
+### Migration compatibility matrix
+
+You can migrate Redis server to Valkey. Migration from Redis stack is not available.
+
+The following table provides migration options depending on the Redis version you run:
+
+| Redis | Valkey |
+|-------|--------|
+| 6.4   | 7.2    |
+| 7.2.x | 7.2.x  |
+| 7.4   | n/a    |
+| 7.0   | 8.0    |
+
+## Migrate a standalone instance
+
+To migrate a standalone Redis server to Valkey, you have the following options:
+
+* [Physical migration](#physical-migration) by copying the most recent on-disk snapshot from the Redis server to Valkey and starting Valkey server with it
+* [Setting up replication](#repication) between Redis and Valkey 
+* [Migrating specific keys](#migrate-specific-keys) 
+
+The example migration steps are provided for Redis 7.2.5 and Valkey version 7.2.6. 
+
+### Physical migration
+
+This is the easiest and fastest migration method. You make a fresh snapshot of your Redis instance and copy it over to Valkey. Valkey reads the data from the snapshot on startup and restores its contents into memory. The tradeoff for this method is the downtime to shutdown Redis and wait for Valkey to load the data. 
+
+1. Make a backup of your Redis instance. 
+
+    1. Connect to Redis and check the number of keys:
```

Comment:
having step 1 then a sub step 1 is confusing. Can you make the substeps alphabetical?